src/components/SavedRecipes/SavedRecipesItem.tsx

```diff
@@ -24,23 +24,32 @@ const SavedRecipesItem = ({ recipe }: SavedRecipeProps) => {
   };
 
   return (
-    <GridItem w="100%" onClick={handleClick}>
-      <Card>
-        <CardBody>
-          <Image
-            boxSize="150px"
-            borderRadius="lg"
-            objectFit="cover"
-            mt="5"
-            src={recipe.recipeImage}
-            alt={recipe.recipeName}
-          />
-          <Stack mt="3">
-            <Center>
-              <Text fontSize="xs" h="7">
-                {recipe.recipeName}
-              </Text>
-            </Center>
+    <GridItem
+      // maxW={{ base: "175px", md: "200px" }}
```

Comment:
NON-BLOCKING: If you don't need these anymore, I'd just delete them. (see line 35 too)